aptos-move/e2e-testsuite/src/tests/emergency_admin_script.rs

```diff
@@ -1,267 +0,0 @@
-// Copyright (c) Aptos
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
what is our point of view about a global, manual upgrade? Do we have a plan on how we'd take care of that?

Reply:
no idea, problem for later. I don't know how things work after we revoke the root account.